src/Aquifer.API/Program.cs

```diff
@@ -22,24 +22,27 @@
 var configuration = builder.Configuration.Get<ConfigurationOptions>()
     ?? throw new InvalidOperationException($"Unable to bind {nameof(ConfigurationOptions)}.");
 
-builder.Services.AddAuthServices(configuration.JwtSettings)
+builder.Services
+    .AddAuthServices(configuration.JwtSettings)
     .AddCors()
     .AddOutputCache()
     .AddMemoryCache()
     .AddApplicationInsightsTelemetry()
     .AddSingleton<ITelemetryInitializer, RequestTelemetryInitializer>()
-    .AddDbContext<AquiferDbContext>(options => options
-        .UseAzureSql(configuration.ConnectionStrings.BiblioNexusDb, providerOptions => providerOptions.EnableRetryOnFailure(3))
-        .EnableSensitiveDataLogging(sensitiveDataLoggingEnabled: builder.Environment.IsDevelopment())
-        .UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking))
+    .AddDbContext<AquiferDbContext>(
+        options => options
+            .UseAzureSql(configuration.ConnectionStrings.BiblioNexusDb, providerOptions => providerOptions.EnableRetryOnFailure(3))
+            .EnableSensitiveDataLogging(builder.Environment.IsDevelopment())
+            .UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking))
     .RegisterModules()
     .Configure<JsonOptions>(options => options.SerializerOptions.Converters.Add(new JsonStringEnumConverter()))
-    .AddHttpLogging(logging =>
```

Comment:
For a single lambda argument I prefer this previous style but there's no way that I can see to make an exception for that particular argument scenario.  The new auto-format style is arguably more readable, anyway.